backend-tests/tests/test_auditlogs.py

```diff
@@ -95,29 +87,77 @@ def test_deployment_create(self, tenant_users):
 
         check_log(res[0], expected)
 
+    def test_user_create(self, tenant_users):
+        user = tenant_users.users[0]
+
+        alogs = ApiClient(auditlogs.URL_MGMT)
+
+        uid = make_user(user.token, "foo@acme.com", "secretsecret")
+        expected = evt_user_create(user, uid, "foo@acme.com")
+
+        time.sleep(0.5)
+        res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
+        assert res.status_code == 200
+        res = res.json()
+        assert len(res) == 1
+
+        check_log(res[0], expected)
+
+    def test_user_delete(self, tenant_users):
+        user = tenant_users.users[0]
+        user_del = tenant_users.users[1]
+
+        alogs = ApiClient(auditlogs.URL_MGMT)
+
+        delete_user(user, user_del.id)
+        expected = evt_user_delete(user, user_del)
+
+        time.sleep(0.5)
+        res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
+        assert res.status_code == 200
+        res = res.json()
+        assert len(res) == 1
+
+        check_log(res[0], expected)
+
+    def test_user_change_role(self, tenant_users):
+        user = tenant_users.users[0]
+        user_change = tenant_users.users[1]
+
+        alogs = ApiClient(auditlogs.URL_MGMT)
+
+        roles = ["RBAC_ROLE_CI"]
+        change_role(user.token, user_change.id, roles)
+        expected = evt_change_role(user, user_change, roles)
+
+        time.sleep(0.5)
+        res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
```

Comment:
Nitpick; just to be safe, maybe we should poll wait on this one? The response time may vary since the POST needs to bounce of workflows as well.
```suggestion
        for _ in range(20):
            time.sleep(0.5)
            res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
            if res.status_code == 200:
                break
```